ietf-optical-impairment-topology.yang

```diff
@@ -399,42 +399,71 @@ module ietf-optical-impairment-topology {
           a specification in a separate equipment catalog";
       }
       container operational {
-        description "amplifier operationnal parameters";
-        leaf actual-gain {
-          type decimal64 {
-            fraction-digits 2;
+        description "amplifier operational parameters";
+        list amplifier-element {
```

Comment:
Hi Aihua, thanks for the review. My understanding is that is not needed to have a defined key in case the list is not a "configuration" list. Here the list is read only.

Reply:
Yes you are right. We do not need keys for read-only list.